openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating openGL reviews."""
+from openpype.hosts.houdini.api import plugin
+from openpype.lib import EnumDef, BoolDef, NumberDef
+
+
+class CreateReview(plugin.HoudiniCreator):
+    """Review with OpenGL ROP"""
+
+    identifier = "io.openpype.creators.houdini.review"
+    label = "Review"
+    family = "review"
+    icon = "video-camera"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        import hou
+
+        instance_data.pop("active", None)
+        instance_data.update({"node_type": "opengl"})
+        instance_data["imageFormat"] = pre_create_data.get("imageFormat")
+        instance_data["keepImages"] = pre_create_data.get("keepImages")
+
+        instance = super(CreateReview, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)
+
+        instance_node = hou.node(instance.get("instance_node"))
+
+        frame_range = hou.playbar.frameRange()
+
+        filepath = "{root}/{subset}/{subset}.$F4.{ext}".format(
+            root=hou.text.expandString("$HIP/pyblish"),
+            subset="`chs(\"subset\")`",  # keep dynamic link to subset
+            ext=pre_create_data.get("image_format") or "png"
+        )
+
+        parms = {
+            "picture": filepath,
+
+            "trange": 1,
+
+            # Unlike many other ROP nodes the opengl node does not default
+            # to expression of $FSTART and $FEND so we preserve that behavior
+            # but do set the range to the frame range of the playbar
+            "f1": frame_range[0],
+            "f2": frame_range[1],
+        }
+
+        override_resolution = pre_create_data.get("override_resolution")
+        if override_resolution:
+            parms.update({
+                "tres": override_resolution,
+                "res1": pre_create_data.get("resx"),
+                "res2": pre_create_data.get("resy"),
+                "aspect": pre_create_data.get("aspect"),
+            })
+
+        if self.selected_nodes:
+            # todo: allow only object paths?
+            node_paths = " ".join(node.path() for node in self.selected_nodes)
+            parms.update({"scenepath": node_paths})
```

Comment:
This is the code I have in mind after the discussion. Not the best one but it works at some point.
```suggestion
        if self.selected_nodes:
            for node in self.selected_nodes:
                path = node.path()
                if node.type().name() == "cam":
                    parms.update({"camera": path})
                else:
                    node_paths = " ".join(path)
                    parms.update({"forceobjects": node_paths})